frontend/src/common/utils/validateDetail.ts

```diff
@@ -0,0 +1,8 @@
+import { ERROR_MESSAGE } from '../constants/errorMessage';
+
+export const validateTextarea = (content: string): string => {
+  if (content.length > 5000) {
+    return ERROR_MESSAGE.CONTENT_LENGTH_TOO_LONG;
+  }
+  return '';
+};
```

Comment:
p4: 반환 타입이 string으로 통일되어서 좋다 👍👍
 반환 타입은 추론이 되어서 명시하지 않아도 될 것 같아! 👍 

```suggestion
export const validateTextarea = (content: string) => {
  if (content.length > 5000) {
    return ERROR_MESSAGE.CONTENT_LENGTH_TOO_LONG;
  }
  
  return '';
};
```

Reply:
[refactor: 타입 추론 되는 반환값 제거](https://github.com/woowacourse-teams/2025-Fit-toring/pull/685/commits/f4f1050bd7025a90caed70759b00aa07791fc714)